system_metrics_collector/test/system_metrics_collector/test_linux_cpu_measurement.cpp

```diff
@@ -14,36 +14,84 @@
 
 #include <gtest/gtest.h>
 
+#include <array>
 #include <chrono>
+#include <functional>
 #include <iostream>
-#include <string>
 #include <memory>
 #include <mutex>
+#include <string>
+
+#include "metrics_statistics_msgs/msg/metrics_message.hpp"
 
 #include "../../src/system_metrics_collector/linux_cpu_measurement_node.hpp"
 #include "../../src/system_metrics_collector/proc_cpu_data.hpp"
 
+using metrics_statistics_msgs::msg::MetricsMessage;
+using metrics_statistics_msgs::msg::StatisticDataPoint;
+using metrics_statistics_msgs::msg::StatisticDataType;
+using moving_average_statistics::StatisticData;
+using moving_average_statistics::STATISTICS_DATA_TYPES;
+using system_metrics_collector::processStatCpuLine;
+
 namespace
 {
-constexpr const char proc_sample_1[] =
-  "cpu  22451232 118653 7348045 934943300 5378119 0 419114 0 0 0\n";
-constexpr const char proc_sample_2[] =
-  "cpu  22451360 118653 7348080 934949227 5378120 0 419117 0 0 0\n";
-constexpr const std::chrono::milliseconds TEST_PERIOD =
+
+constexpr const char TEST_NODE_NAME[] = "test_measure_linux_cpu";
+constexpr const char TEST_TOPIC[] = "test_cpu_measure_topic";
```

Comment:
Use std::string

Reply:
Unable to use `constexpr` `std::string` in C++14.